### PR TITLE
interactivity analytics

### PR DESCRIPTION
**Pull request by RobertoUa**
@shamansir like this

### CODE REVIEW

File: src/anm/analytics.js [hunk 1]
```diff
@@ -53,6 +53,20 @@ Analytics.prototype.trackPlayer = function trackPlayer(name) {
...
+    var opts = {
+        viewId: player.viewId,
+        projectId: player.anim.meta._anm_id,
+        time: player.state.time,
+        interactivity: {
+            path: path,
+            type: type
+        }
+    };
+    if (utils.is.num(time)) opts.interactivity.time = time;
```
Comment (shamansir) [inline src/anm/analytics.js]: why not this: 

```javascript
var opts = {
    // ...
    time:  utils.is.num(time) ? time : player.state.time,
    // ...
}
```

Comment (shamansir) [inline src/anm/analytics.js]: uh, it's different `time`, go it. why two of them, then? :)

Comment (RobertoUa) [inline src/anm/analytics.js]: actually that's what I was going to do, but I forgot.

Comment (shamansir) [inline src/anm/analytics.js]: ok, good!

**Pull request merged**